src/transport/stub/mod.rs

```diff
@@ -1,18 +1,20 @@
-//! The stub transport only logs message envelope and drops the content. It can be useful for
-//! testing purposes.
+//! The stub transport logs message envelopes as well as contents. It can be useful for testing
+//! purposes.
 //!
-//! #### Stub Transport
+//! # Stub Transport
 //!
-//! The stub transport returns provided result and drops the content. It can be useful for
-//! testing purposes.
+//! The stub transport logs message envelopes as well as contents. It can be useful for testing
+//! purposes.
+//!
+//! # Examples
 //!
 //! ```rust
 //! # #[cfg(feature = "builder")]
 //! # {
 //! use lettre::{transport::stub::StubTransport, Message, Transport};
 //!
 //! # use std::error::Error;
-//! # fn main() -> Result<(), Box<dyn Error>> {
+//! # fn try_main() -> Result<(), Box<dyn Error>> {
```

Comment:
What's the reason for this change?

Reply:
The test didn't do anything before as the default generated `fn main` by rustdoc isn't async.